### PR TITLE
New event_detail template

### PR DESCRIPTION
**Pull request by dearkimlow**
Created new event_detail template based on the existing event_form template. Change details:

1. Removed code specifically marked as not read-only, including is_census_equipped and is_ada_compliant.
2. Replaced instances of form, label, input, textarea, etc. with div, h3, or p and updated classes for CSS styling.
3. Commented out "(optional)" from any form labels.
4. Did NOT update code for event_type or form.languages.field.choices, since I'm not familiar enough with Django to figure out the values to display for those.

Also updated views.py to point ShowEvent to the new template. Please confirm that this was done correctly!

### CODE REVIEW

**Comment by mikeubell** [comment]
seems to be some missing fields and End Time has a problem.
Screen Shot 2020-01-18 at 11 24 05 AM

File: census/templates/census/event_detail.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                          <p>value="{{form.end_datetime.value| date:'Y-m-d H:i'}}</p>
+                      </div>
+                    </div>
+                    {% if enable_recurrence %}
+                    <span class="usa-hint line-height-body-1 font-body-2xs">{{ form.recurrences.help_text }}</span>
+                    <div class="grid-row">
+                        {{ form.recurrences }}
+                    </div>
+                    {% else %}
+                    <input {{ readonly }} type="hidden" name="{{ form.recurrences.name }}" value="" />
```
Comment (mikeubell) [inline census/templates/census/event_detail.html]: I believe you need to remove all references to {{ readonly }}

**Comment by dearkimlow** [comment]
I think those missing fields are the new city and zip code fields that were recently added in issue #130, so I'm realizing that I can't just add my code on my page and make it work. What's the best way to handle this? Do I remove the city and zip code info for now, edit everything else, and then make those edits after this branch is finished/resolved/merged and I create a new clone? Or is there some way to grab the relevant files and work them into my branch?

In the meantime I'll make the other changes. Thanks for reviewing!

**Comment by dearkimlow** [comment]
Just realized that those missing city and zip code fields are useless on this page since the full address is already shown. So I removed those fields after all. I pushed all of my edits; let me know if more changes are needed?

**Comment by mikeubell** [comment]
This looks better.   I think leaving off city and zip is ok.  I do think the font on the data is a bit small relative to the titles, since we are displaying, rather than inputting the data should be bigger..  Also there is lots of white space, is there some way to tighten it up.

**Comment by mikeubell** [comment]
Thinking about it a bit more.  It might look better if rather than having "Event Details" as the title, just use the title in the large font (without the Title: label).  We can probably drop the description title too as that would be redundant.  I think I would reverse Location Name and Address and move the Start/End times above the contact info.  While we are at we probably don't to label the email and phone number either. Something like:

### Census Information Session
Find out what you need to know to promote the Census 2020 in your community
**Location:**
City Hall
1 Frank Ogawa Plaza Oakland, CA
**Time:**
2020-01-20 18:00  to 2020-01-20 20:00
**Contact:**
OpenOakland
Michael Ubell
mike.ubell@gmail.com
510-555-1212

**Comment by dearkimlow** [comment]
Styles have been updated, including some changes in how event type is shown. I also added the languages supported field, since I think this info would be helpful for people who are more comfortable using other languages. However, I'm not sure if my code for outputting this value is correct. Do I need to make any modifications to show multiple language values?

**Comment by mikeubell** [comment]
Looking good!  I did get this under the address:
{ form.location.value|default_if_none:''}}

**Pull request merged**